### PR TITLE
updated purescript example for compiler version >= 0.7.4.1

### PR DESCRIPTION
**Pull request by Thimoteus**

### CODE REVIEW

File: purescript.html.markdown [hunk 1]
```diff
@@ -2,194 +2,207 @@
...
 
 PureScript is a small strongly, statically typed language compiling to Javascript.
 
 * Learn more at [http://www.purescript.org/](http://www.purescript.org/)
-* Documentation: [http://docs.purescript.org/en/latest/](http://docs.purescript.org/en/latest/)
+* Documentation: [http://pursuit.purescript.org/](http://pursuit.purescript.org/)
 * Book: Purescript by Example, [https://leanpub.com/purescript/](https://leanpub.com/purescript/)
 
-```haskell
+```purescript
```
Comment (vendethiel) [inline purescript.html.markdown]: I'm not sure our lexer has purescript.

File: purescript.html.markdown [hunk 2]
```diff
@@ -2,194 +2,207 @@
...
 
 --
 -- 1. Primitive datatypes that corresponds to their Javascript
 -- equivalents at runtime.
 
+> import Prelude
 -- Numbers
-1 + 7*5 :: Number -- 36
+> 1.0 + 7.2*5.5 :: Number -- 40.6
```
Comment (vendethiel) [inline purescript.html.markdown]: I'm not sure I see the point of all these `>`. They're just adding clutter, and making syntax highlighting worse...

**Comment by Thimoteus** [comment]
Ahh, I know Github supports purescript syntax highlighting so I added it in. The `> `'s were added to make it clear that lines can be run in the REPL.

**Comment by Thimoteus** [comment]
OK, I removed the `>` symbols and added a line explaining what can be run in the REPL. Also switched back to haskell highlighting.

**Comment by Thimoteus** [comment]
Any further comments @vendethiel ?

**Comment by vendethiel** [comment]
Yes: thank you! :cake:

**Pull request merged**